src/shogun/base/SGObject.h

```diff
@@ -287,6 +287,13 @@ class CSGObject
 	 */
 	Version* get_global_version();
 
+	/**
+	 * Return the description of a registered parameter given its name
+	 * @param name parameter's name
+	 * @return description of the parameter as a string
+	 */
+	std::string get_description(const std::string& name) const;
```

Comment:
@karlnapf Since there was no way to extract the description of a parameter easily, I moved this method from the SWIG interface to the actual `SGObject.h` (which was originally added by @gf712). I guess it makes sense since it is wayy much simpler now. For instance, if we look at C++:
```c++
object.get_parameters().find(name)->second.get()->get_properties()->get_description()
```
versus:
```c++
object.get_description(name)
```

Reply:
ok.... we kinda saw it coming :D